controllers/metal3.io/hostfirmwarecomponents_controller.go

```diff
@@ -223,9 +223,7 @@ func (r *HostFirmwareComponentsReconciler) updateHostFirmware(info *rhfcInfo, co
 
 	// Update Status if has changed
 	if dirty {
-		info.log.Info("Status for HostFirmwareComponents changed")
-		info.hfc.Status = *newStatus.DeepCopy()
```

Comment:
Without this you're no longer writing the components read from ironic.

Reply:
Ok